packages/hardhat/contracts/YourContract.sol

```diff
@@ -88,7 +90,7 @@ contract YourContract is Ownable {
     /// @param _builder Address of the target builder
     /// @return Amount of funds unlocked at this point in time
     function unlockedBuilderAmount(address _builder) public view returns (uint128) {
-        BuilderStreamInfo memory builderStream = streamedBuilders[_builder];
+        BuilderStreamInfo storage builderStream = streamedBuilders[_builder];
```

Comment:
don't need storage here since we update the struct with the mapping

Reply:
Loading the entire struct into `memory` incurs a gas cost equivalent to the memory expansion of holding all the values of the struct in memory. Using a storage pointer is about 100 gas cheaper in your current master contract (just changing memory to storage). 

In this implementation it's only about 30 gas cheaper due to needing to cast to a uint128 and with the struct needing 32 bytes less memory.